frappe/core/doctype/communication/email.py

```diff
@@ -77,7 +147,9 @@ def make(doctype=None, name=None, content=None, subject=None, sent_or_received =
 		"read_receipt":read_receipt,
 		"has_attachment": 1 if attachments else 0,
 		"communication_type": communication_type,
-	}).insert(ignore_permissions=True)
+	})
+	comm.flags.skip_add_signature = True
```

Comment:
the internal method should always try to add signature, right?
```suggestion
	comm.flags.add_signature = True
```
?

Shouldn't we move this to the whitelisted `make` method to explicitly avoid adding signature?